### ISSUE TITLE
Do we really need 4 fill value aliases?

### ISSUE THREAD

**Issue by mhidas** [aodn/aodn-netcdf-tools#19]
Currently the code allows the fill value for a variable to be specified in an attribute with any of these names 
```python
self.fill_aliases = {'fill_value', 'missing_value', 'FillValue', '_FillValue'}
```
It doesn't matter which you use, this is treated as a special attribute and passed as the `fill_value` option to [`Dataset.createVariable()`](http://unidata.github.io/netcdf4-python/#netCDF4.Dataset.createVariable). The  actual attribute name written to the netCDF file is always  "_FillValue", which is the standard one specified by [CF](http://cfconventions.org/cf-conventions/v1.6.0/cf-conventions.html#missing-data) and the NetCDF User Guide ([NUG](https://www.unidata.ucar.edu/software/netcdf/docs/attribute_conventions.html)) (Both of these also mention "missing_value", but it has a slightly different definition, is allowed to be a vector, and at least according to CF [Appendix A](http://cfconventions.org/cf-conventions/v1.6.0/cf-conventions.html#attribute-appendix) it is actually deprecated by the NUG.)

So to avoid confusion and simplify the code, I think it would make sense to only treat "_FillValue" as a special attribute, i.e. it should be the way to specify what the fill value should be.

**Comment by mhidas** [comment]
"missing_value" may have a similar meaning, but is **not equivalent** to "_FillValue", according to the [NUG](https://www.unidata.ucar.edu/software/netcdf/docs/attribute_conventions.html)
> _FillValue
> The _FillValue attribute specifies the fill value used to pre-fill disk space allocated to the variable. Such pre-fill occurs unless nofill mode is set using nc_set_fill(). The fill value is returned when reading values that were never written. If _FillValue is defined then it should be scalar and of the same type as the variable.

> missing_value
This attribute **is not treated in any special way by the library or conforming generic applications**, but is often useful documentation and may be used by specific applications. The missing_value attribute **can be a scalar or vector** containing values indicating missing data.

Based on that, I don't think we should treat an attribute named "missing_value" in any special way.

I think we should only accept "_fill_value" or "_FillValue"

**Comment by mhidas** [comment]
Fixed in #36
